docs/5-data-diffing/connect-to-warehouse.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Connect to Warehouse
+---
+
+## Recce OSS
+If you use the open source setup, you can connect to your warehouse the same way as dbt. No additional settings need to be configured. Recce uses the dbt library with your profiles to connect to your warehouse.
```

Comment:
If you use the open source setup, you can connect to your warehouse the same way as dbt. Recce supports any warehouse that dbt supports. No additional settings need to be configured; Recce uses the dbt library with your profiles to connect to your warehouse.

I want to say it clear that in OSS,  "Recce supports any warehouse that dbt supports."